### ISSUE TITLE
UnboundLocalError: local variable 'found_ntaxa' referenced before assignment

### ISSUE THREAD

**Issue by xrotwang** [dlce-eva/python-nexus#1]
I just ran into this exception after upgrading python-nexus for dplace:

```
    reader = NexusReader(file_name)
  File "/home/robert/venvs/dplace/local/lib/python2.7/site-packages/nexus/reader.py", line 674, in __init__
    return self.read_file(filename)
  File "/home/robert/venvs/dplace/local/lib/python2.7/site-packages/nexus/reader.py", line 706, in read_file
    self._read(handle)
  File "/home/robert/venvs/dplace/local/lib/python2.7/site-packages/nexus/reader.py", line 751, in _read
    self._do_blocks()
  File "/home/robert/venvs/dplace/local/lib/python2.7/site-packages/nexus/reader.py", line 682, in _do_blocks
    self.blocks[block].parse(data)
  File "/home/robert/venvs/dplace/local/lib/python2.7/site-packages/nexus/reader.py", line 146, in parse
    if found_ntaxa != self.ntaxa:
UnboundLocalError: local variable 'found_ntaxa' referenced before assignment
```

**Comment by SimonGreenhill** [comment]
@xrotwang - can you send me the nexus file?

**Comment by xrotwang** [comment]
It's one of the files in dplace-data. Can check which tomorrow.
Am 02.02.2016 22:40 schrieb "Simon J Greenhill" notifications@github.com:

**Comment by SimonGreenhill** [comment]
Fixed in v1.34 -- I'll upload to PYPI later.
